packages/pdk-nag/src/packs/aws-prototyping-rules.ts

```diff
@@ -0,0 +1,29 @@
+/*! Copyright [Amazon.com](http://amazon.com/), Inc. or its affiliates. All Rights Reserved.
+SPDX-License-Identifier: Apache-2.0 */
+import { NagMessageLevel, rules } from "cdk-nag";
+
+export let PackName = "AwsPrototyping";
```

Comment:
missing jsdocs

Reply:
Fixed in [43523519d4a565d9d12cb76e4e1335a25fcea7b2](https://github.com/aws/aws-prototyping-sdk/pull/213/commits/43523519d4a565d9d12cb76e4e1335a25fcea7b2)